### DISCUSSION TITLE
Add some sort of basic catridge "store" or "hub"

### DISCUSSION THREAD

**Discussion by darltrash** [nesbox/TIC-80#1299] [Ideas]
It would be cool if you could check new catridges (and download them) from the tic80 program itself, like what surf does but online.

**Comment by Anrock** [comment]
But you already can at https://tic80.com/play? Just click on a cartridge and there is download button on top of the player.

**Reply by darltrash** [reply]
That isnt exactly the same as adding one onto the tic80 executable itself, consider that tic80 can also be used for rpi-based projects

**Comment by Tele256** [comment]
Pretty sure Surf **is** online. If it wasn't online, it would work even without internet and it wouldn't update as people post more stuff to https://tic80.com/play. It currently does both of those things.
